src/forms/formHelpers.js

```diff
@@ -471,52 +490,61 @@ const CashFlowRow = function ({ generic, timeState, setClientProperty, children,
    *     different?
    */
   var baseVal   = timeState[ generic ],
-      classes   = [ time, type, 'cashflow-column' ],
       baseProps = {
-        store:    updateClient,
-        validate: isPositiveNumber,
-        format:   toMoneyStr
+        name:       generic,
+        className:  'cashflow-column',
+        store:      updateClient,
+        validation: isPositiveNumber,
+        format:     toMoneyStr,
+        onBlur:     function () { return true; }
       };
 
   return (
-    <Form.Field inline className={'cashflow'}>
+    <CashFlowContainer label={children} validRow={validRow || true} message={message || null}>
```

Comment:
`validRow || true` will always be truthy.

Reply:
The only reason this doesn't cause a problem is because `<RowMessage>` checks both `validRow` and `message` before showing the label.

Reply:
It will right now. I was making this in hopeful anticipation of getting rid of `<MonthlyCashFlowRow>` and adapting plain old `<CashFlowRow>` to be more flexible. Nothing I've worked out so far works very well for that, though. As for `<RowMessage>` catching it, if `validRow === true`, no message would show anyway.

Still, may be time to get rid of it.

Reply:
I think `validRow` will still always need to be true for the sake of clarity.

Reply:
Reviewing the code I was a little off base as this isn't actually a problem, even if `val || true` is a little worrying. I guess I thought this was being used by `<RentShareField>`, but that's the other one.